bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -23,294 +23,294 @@ spec:
                 securityContext:
-                  allowPrivilegeEscalation: false
-              imagePullPolicy: Always 
-              imagePullSecrets:
-              - name: search-pull-secret
-              securityContext:
-                runAsNonRoot: true
-              serviceAccountName: search-v2-operator
-              terminationGracePeriodSeconds: 10
+                  runAsNonRoot: true
+                serviceAccountName: search-v2-operator
+                terminationGracePeriodSeconds: 10
       permissions:
-      - rules:
-        - apiGroups:
-          - ""
-          resources:
-          - configmaps
-          verbs:
-          - get
-          - list
-          - watch
-          - create
-          - update
-          - patch
-          - delete
-        - apiGroups:
-          - coordination.k8s.io
-          resources:
-          - leases
-          verbs:
-          - get
-          - list
-          - watch
-          - create
-          - update
-          - patch
-          - delete
-        - apiGroups:
-          - ""
-          resources:
-          - events
-          verbs:
-          - create
-          - patch
-        serviceAccountName: search-v2-operator
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: search-v2-operator
     strategy: deployment
   installModes:
-  - supported: true
-    type: OwnNamespace
-  - supported: false
-    type: SingleNamespace
-  - supported: false
-    type: MultiNamespace
-  - supported: true
-    type: AllNamespaces
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
   keywords:
-  - Advanced Cluster Management
-  - acm
-  - Open Cluster Management
-  - ocm
-  - odyssey
-  - search
-  - stolostron
+    - Advanced Cluster Management
+    - acm
+    - Open Cluster Management
+    - ocm
+    - odyssey
+    - search
+    - stolostron
   links:
-  - name: Search Operator
-    url: https://github.com/stolostron/search-v2-operator
+    - name: Search Operator
+      url: https://github.com/stolostron/search-v2-operator
   maturity: alpha
   provider:
```

Comment:
@dislbenn We dont have any changes to this file right ? Looks like some formatting difference .. thats it ?

Reply:
The format was changed when the script was executed; however, it seems like the only thing that was updated was the spacing.